### PR TITLE
Gradle build

### PR DESCRIPTION
**Pull request by broady**
@brendankenny PTAL

Fixes #1

### CODE REVIEW

**Comment by broady** [comment]
Just realised this doesn't have the file wrapped in a function. Will add that.

**Comment by broady** [comment]
Added wrapper task.

@markmcd PTAL too?

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,43 @@
+plugins {
+  id "com.eriwen.gradle.js" version "1.12.1"
```
Comment (markmcd) [inline build.gradle:2]: Through what black magic does this work? Does gradle 2 have a built in repository somewhere?

**Comment by markmcd** [comment]
Nice. LGTM.

File: build.gradle (hunk 1 above)
Comment (broady) [inline build.gradle:2]: yeah, gradle 2.1 adds that - it's new!

**Pull request merged**